wiki/plugins/macros/mdx/macro.py

```diff
@@ -1,106 +1,255 @@
 # -*- coding: utf-8 -*-
+
```

Comment:
A bunch of blank lines are showing up in the diff. These are probably due to Windows line breaks. Please make sure your editor is set to Unix line breaks.